Applications/AlexNet/gen_input/convert.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+#
+# @file	  convert.py
+# @date	  12 Oct 2020
+# @brief  This is script to convert cifar100 data set for vgg model
+# @see    https://github.com/nnstreamer/nntrainer
+# @author Jijoong Moon <jijoong.moon@samsung.com>
+# @bug    No known bugs except for NYI items
+#
+# python3 convert.py path_for_cifar100 [train | val]
+#  - trainingSet.dat for training : 100 images x 100 classes
+#  - valSet.dat for validation : 20 images x 100 classes
+#
+
+import os
+import sys
+import imageio
+import numpy as np
+import random
```

Comment:
Opinion)
In my humble opinion, simply copying the `convert.py` from `Application/VGG/get_input` folder is not a good idea. The same files exist and it causes a conflict issue when modifying one of them.

```python
import sys
import sys.path.insert(2, '...')
import convert


if __name__ == "__main__":
    ...
```

Reply:
OK.